frontend/packages/dev-console/src/hooks/useAddActionsAccessReviews.ts

```diff
@@ -53,7 +60,7 @@ export const useAddActionsAccessReviews = (
           });
       }
     });
-  }, [accessReviewResults, addActionExtensions, namespace]);
+  }, [accessReviewResults, addActionExtensions, initialState, namespace]);
```

Comment:
do we need to `initialState` here in deps?

Reply:
no, it doesn't have to be a dependency. removed it.